test/controllers/actions/DataRetrievalActionSpec.scala

```diff
@@ -35,6 +35,7 @@ class DataRetrievalActionSpec extends SpecBase with MockitoSugar with ScalaFutur
   }
 
   "Data Retrieval Action" when {
+    val nino = "AB123456A"
```

Comment:
fakeNino now in spec base so can remove this val and use fakeNino throughout this file and all other tests requiring nino